kubernetes/test-infra/gke/README.md

```diff
@@ -20,11 +20,6 @@ See https://cloud.google.com/kubernetes-engine/versioning-and-upgrades#versions_
 ## Exporting K8S variables
 
 ```
-export KUBE_HOST=https://$(terraform output endpoint)
-export KUBE_USER=$(terraform output username)
-export KUBE_PASSWORD=$(terraform output password)
-export KUBE_CLIENT_CERT_DATA="$(terraform output client_certificate_b64 | base64 -d -)"
-export KUBE_CLIENT_KEY_DATA="$(terraform output client_key_b64 | base64 -d -)"
-export KUBE_CLUSTER_CA_CERT_DATA="$(terraform output cluster_ca_certificate_b64 | base64 -d -)"
-export GOOGLE_ZONE=$(terraform output zone)
+export KUBECONFIG="${KUBECONFIG}:$(terraform output kubeconfig_path)"
+export GOOGLE_ZONE=$(terraform output google_zone)
```

Comment:
This one is supposed to be an input to the templates, so most of the time it's already set. But let's keep it in here for clarity and cases where a default is used.

More importantly, KUBE_CTX might need to be set as well, to the value of the relevant context from Kubeconfig.
Would you mind adding an output for that one too?

Thanks for taking care of this.

Reply:
Hmm, the template is using a datasource right now: `${data.google_compute_zones.available.names[0]}`: https://github.com/terraform-providers/terraform-provider-kubernetes/blob/v1.5.0/kubernetes/test-infra/gke/main.tf#L24

Using `export KUBECONFIG="$(terraform output kubeconfig_path)"` avoids having to set `KUBE_CTX` as there's only a single cluster and context in that file.